pkg/resource/events-logger-config/grafana-agent-config.go

```diff
@@ -60,39 +44,37 @@ func generateGrafanaAgentConfig(lc loggedcluster.Interface, defaultWorkloadClust
 		GrafanaAgentInnerConfig: grafanaAgentInnerConfig,
 	}
 
-	err = grafanaAgentTemplate.Execute(&values, data)
+	err = grafanaAgentConfigTemplate.Execute(&values, data)
 	if err != nil {
 		return "", err
 	}
 
 	return values.String(), nil
 }
 
-func generateGrafanaAgentInnerConfig(lc loggedcluster.Interface, defaultWorkloadClusterNamespaces []string) (string, error) {
+func generateGrafanaAgentInnerConfig(lc loggedcluster.Interface, defaultNamespaces []string) (string, error) {
 	var values bytes.Buffer
 
 	data := struct {
 		ClusterID          string
 		Installation       string
 		InsecureSkipVerify string
 		SecretName         string
+		SecretNamespace    string
 		ScrapedNamespaces  string
 	}{
 		ClusterID:          lc.GetClusterName(),
 		Installation:       lc.GetInstallationName(),
 		InsecureSkipVerify: fmt.Sprintf("%t", lc.IsInsecureCA()),
-		SecretName:         fmt.Sprintf("%s-%s", lc.GetClusterName(), common.GrafanaAgentExtraSecretName()),
-		ScrapedNamespaces:  common.FormatScrapedNamespaces(lc, defaultWorkloadClusterNamespaces),
+		SecretName:         eventsloggersecret.GetEventsLoggerSecretName(lc),
+		SecretNamespace:    lc.GetAppsNamespace(),
```

Comment:
App namespace will bé org-xxx but we expect kube-system so enforce it in template

Reply:
Yeah that's already done (by you I think) :)